modules/imgcodecs/CMakeLists.txt

```diff
@@ -1,3 +1,7 @@
+if(WINRT)
+  ocv_module_disable(imgcodecs)
+endif()
```

Comment:
Ok, sounds reasonable.
Separate patch will be good, because this patch is big enough.